.github/workflows/_buildpacks-release-detect.yml

```diff
@@ -1,26 +0,0 @@
-name: _buildpacks-release-detect
```

Comment:
The references to the old workflows can now be removed from the README:
https://github.com/heroku/languages-github-actions#workflows

(Also happy for the README change to wait until another PR, if you'd prefer to document the new workflow at the same time as removing the old ones, and don't want to add it in this PR)

Reply:
I was thinking of updating the docs as a separate PR but it makes more sense to include them with these removals.